source/administration/encryption.rst

```diff
@@ -14,7 +14,15 @@ Encryption-in-transit
 
 Mattermost supports TLS encryption including AES-256 with 2048-bit RSA on all data transmissions between Mattermost client applications and the Mattermost server. You may either set up TLS on the Mattermost Server or install a proxy such as NGINX and set up TLS on the proxy. Refer to our `configuration guide for more details <https://docs.mattermost.com/install/config-tls-mattermost.html>`__.
 
-Connections to Active Directory/LDAP can `optionally be secured with TLS or stunnel <https://docs.mattermost.com/administration/config-settings.html#id11>`__.
+Connections to Active Directory/LDAP can `optionally be secured with TLS or stunnel <https://docs.mattermost.com/administration/config-settings.html#id9>`__.
+
+Gossip Encryption (Experimental):
+-----------------------
+
+In a High Availability mode, Mattermost supports encryption of cluster data in-transit when using the gossip protocol.  
+
+The encryption uses AES-256 by default, and it is not configurable. However, it is possible to manually set the value in the `Systems` table to AES-128, AES-192, or AES-256.
```

Comment:
This gives the impression that one can set the option of which encryption algorithm to use in the `Systems` table. But that's actually not the case. What we can set in the table is the encryption key. And the length of the encryption key in turn determines the encryption algorithm.

Reply:
@agnivade Updated. please let me know if that resolves your concern. Also, please let me know if you suggest better wording.